### DISCUSSION TITLE
My button is very small. Much smaller than it appears in the preview version and i can't enlarge it.

### DISCUSSION THREAD

**Discussion by UhtredofBamburgh** [woocommerce/woocommerce#46028] [Checkout Flow]
My button is very small. Much smaller than it appears in the preview version and i can't enlarge it. 

<img width="792" alt="Screenshot 2024-03-27 at 9 39 45 PM" src="https://github.com/woocommerce/woocommerce/assets/129151419/dc20c2bf-5ae3-46fd-ba36-d5255e47cb39">
<img width="1373" alt="Screenshot 2024-03-27 at 9 39 57 PM" src="https://github.com/woocommerce/woocommerce/assets/129151419/1f9fadaa-5b9a-4766-83e7-3fde9ad24226">

**Comment by opr** [comment]
Hi there @UhtredofBamburgh thanks for starting this discussion. What theme are you using? It's possible that this theme is overriding the styles of the button on the front end. Can you try with a default theme, such as Twenty Twenty-Four and see if the problem persists?
